src/codec_aom.c

```diff
@@ -555,11 +584,48 @@ static avifBool doesLevelMatch(int width, int height, int levelWidth, int levelH
     return lumaPels <= levelLumaPels && width <= levelWidth * levelDimMult && height <= levelHeight * levelDimMult;
 }
 
+// Quality (q) to quantizer (qp) formula for tune=iq (Image Quality), expressed as a look-up table for more clarity.
+// The x axis of the table represents the ones digit, while the y axis represents the tens digit
+// of the quality value [0-100], which is then mapped to a quantizer value [0-63].
+// The formula below is a piecewise linear function. Each segment was empirically selected to correct for the
+// non-linear bitrate increase from encoding with tune=iq (relative to tune=ssim), with a goal to make tune=iq behave
+// like tune=ssim at matching quality levels, with an overall smaller (but still predictable) file size and a similar
```

Comment:
Nit/optional: "behave like tune=ssim" could be misinterpreted to mean tune=iq operates like tune=ssim, but I think you are referring to the encoding time? Would be good to clarify. I will also think about how to rephrase this.